scripts/gdb.py

```diff
@@ -174,6 +178,28 @@ def set_state(self, registers):
     for r, v in registers.items():
       gdb.execute("set ${} = {}".format(r,v))
 
+class Sgx_Add_Symbol_File (gdb.Command):
+  """
+  Given any address in the enclave and ELF path, load enclave's symbols.
+  sgx-add-symbol-file <ADDR_IN_ENCLAVE | auto> <ELF_PATH>
+  Specifing address as auto is equivalent to specifying it in $rbx.
```

Comment:
"specifying it in $rbx" doesn't make sense. You can say "specifying $rbx"

Reply:
s/Specifing/Specifying/

Reply:
Corrected in recent push.

Reply:
Corrected in recent push.